### DISCUSSION TITLE
DWML is not working

### DISCUSSION THREAD

**Discussion by vbguyny** [weather-gov/api#321] [Q&A]
According to the FAQ (https://weather-gov.github.io/api/general-faqs), I should still be able to get a weather forecast in the previous DWML format.  However, it does not work for me.

Request:
```
curl -X GET "https://api.weather.gov/gridpoints/LWX/96,70/forecast?units=us" -H "accept: application/vnd.noaa.dwml+xml"
```

Response:
```
{
    "correlationId": "e144413d-f195-4f9c-b6fe-d57a23026ef1",
    "title": "Unexpected Problem",
    "type": "https://api.weather.gov/problems/UnexpectedProblem",
    "status": 500,
    "detail": "An unexpected problem has occurred.",
    "instance": "https://api.weather.gov/requests/e144413d-f195-4f9c-b6fe-d57a23026ef1"
}
```

**Comment by mcarroll316** [comment]
Is there a new endpoint up that can be used as they're phasing the old one out?  I specifically use this one:

https://api.weather.gov/gridpoints/LWX/97,69

...but it stopped being updated last evening (7/8 @ 20:00)

Furthermore, aside from the endpoint changing, will the format change?  I just recently build my app to use JSON (from the link I provided) and really hoping it remains the same.

**Comment by scadergit** [comment]
The recent API upgrade introduced a bug to the DWML format. I would recommend reporting this as an outage so that it can be prioritized.

**Comment by mcarroll316** [comment]
Thank you **scadergit** - I just emailed NOAA with the information.

**Comment by kcajf** [comment]
I'm seeing this too. Any update @mcarroll316 ?

**Comment by vbguyny** [comment]
The outage was related to a server migration on NOAA's end.  My site has been working since they resolve the issue several weeks ago.

**Comment by kcajf** [comment]
Are you still able to run the `curl` example pasted above? If I run it, I get:
```
curl -X GET "https://api.weather.gov/gridpoints/LWX/96,70/forecast?units=us" -H "accept: application/vnd.noaa.dwml+xml"
{
    "correlationId": "3431ae8e",
    "title": "Unexpected Problem",
    "type": "https://api.weather.gov/problems/UnexpectedProblem",
    "status": 500,
    "detail": "An unexpected problem has occurred.",
    "instance": "https://api.weather.gov/requests/3431ae8e"
}
```

**Comment by vbguyny** [comment]
I haven't tried since NOAA resolved the DWML issue.  I suggest reaching out to NOAA directly.
